api_tester/lib/models/customer_info_wrapper_api_test.dart

```diff
@@ -0,0 +1,75 @@
+import 'package:purchases_flutter/object_wrappers.dart';
+
+// ignore_for_file: unused_element
+// ignore_for_file: unused_local_variable
+class _CustomerInfoApiTest {
+  void _checkFromJsonFactory(Map<String, dynamic> json) {
+    CustomerInfo customerInfo = CustomerInfo.fromJson(json);
+  }
+
+  void _checkToJson(CustomerInfo customerInfo) {
+    Map<String, dynamic> json = customerInfo.toJson();
+  }
+
+  void _checkConstructor(
+      EntitlementInfos entitlements,
+      Map<String, String> allPurchaseDates,
+      List<String> activeSubscriptions,
+      List<String> allPurchasedProductIdentifiers,
+      List<StoreTransaction> nonSubscriptionTransactions,
+      String firstSeen,
+      String originalAppUserId,
+      Map<String, String?> allExpirationDates,
+      String requestDate,
+      String? latestExpirationDate,
+      String? originalPurchaseDate,
+      String? originalApplicationVersion,
+      String? managementURL
+      ) {
+    CustomerInfo customerInfo = CustomerInfo(
+        entitlements,
+        allPurchaseDates,
+        activeSubscriptions,
+        allPurchasedProductIdentifiers,
+        nonSubscriptionTransactions,
+        firstSeen,
+        originalAppUserId,
+        allExpirationDates,
+        requestDate
+    );
+    customerInfo = CustomerInfo(
```

Comment:
Note how I'm checking the constructor twice, once with the required parameters, and another one with the optional parameters. I thought it was a bit weird that the optional parameters are always named and the required parameters are positional... But I think it's not really a problem